docs/en/observability/synthetics-lightweight.asciidoc

```diff
@@ -55,28 +66,161 @@ heartbeat.scheduler:
   limit: 10
 ----
 
-[discrete]
 [[synthetics-monitor-options]]
-= Configuration options
-
-You can find a complete reference of available configuration options in
-the {heartbeat} documentation.
-
-[NOTE]
-====
-While the most commonly used {heartbeat} configuration options are supported,
-some configuration options are not yet supported in {project-monitors} and the {synthetics-app}.
-====
-
 There are some common monitor configuration options that are the same for all monitors.
-For a complete list, refer to {heartbeat-ref}/monitor-options.html[Common monitor options].
+For a complete list, refer to <<synthetics-lightweight-common-options>>.
 
 Each monitor type has additional configuration options that are specific to that monitor type.
 Refer to:
 
-* {heartbeat-ref}/monitor-http-options.html[HTTP monitors]
-* {heartbeat-ref}/monitor-tcp-options.html[ICMP monitors]
-* {heartbeat-ref}/monitor-tcp-options.html[TCP monitors]
+* <<synthetics-lightweight-http>>
+* <<synthetics-lightweight-icmp>>
+* <<synthetics-lightweight-tcp>>
+
+The `tcp` and `http` monitor types both support SSL/TLS and some proxy settings.
+
+[discrete]
+[[synthetics-lightweight-common-options]]
+== Common options
+
+You can specify the following options when defining a synthetic monitor in any location.
+These options are the same for all monitors. Each monitor type has additional configuration
+options that are specific to that monitor type.
+
+// Reference table
+include::synthetics-reference/lightweight-config/common.asciidoc[]
+
+[discrete]
+[[synthetics-lightweight-http]]
+== HTTP options
+
+// Is this still relevant?
+The options described here configure Heartbeat to connect via HTTP and
+optionally verify that the host returns the expected response.
```

Comment:
Should the mention of "Heartbeat" here be removed? If yes, what's the right alternative?

Reply:
I think we should generally replace all occurrences of Heartbeat with Synthetics except where absolutely necessary, if perhaps describing literally using heartbeat itself. When working with elastic-agent or the service we should avoid it